src/faldbt/project.py

```diff
@@ -436,6 +487,16 @@ def _get_models_with_keyword(self, keyword) -> List[DbtModel]:
             filter(lambda model: keyword in model.meta, self._faldbt.list_models())
         )
 
+    def _map_tests_to_models(self, models: List[DbtModel], tests: List[DbtTest]) -> List[DbtModel]:
+        for model in models:
+            model_status = self.get_model_status(model)
```

Comment:
curious about having to do `self.get_model_status(model)` but having a `model.set_status(...)`

If we have a `set_status`, shouldn't we also have a `get_status`?

Reply:
Makes sense